src/app/components/BibPage/components/BibItems.jsx

```diff
@@ -0,0 +1,40 @@
+import PropTypes from 'prop-types';
+import React, { useMemo } from 'react';
+import { useBib } from '../../../context/Bib.Provider';
+import { isElectronic } from '../../../utils/utils';
+import itemsContainerModule from '../../Item/ItemsContainer';
+
+const ItemsContainer = itemsContainerModule.ItemsContainer;
+
+const BibItems = ({ items, keywords, location }) => {
+  const { bib, bibId } = useBib();
+
+  // TODO: useMemo is useless here
```

Comment:
As a TODO for anyone to get to in the future really. It's merely a note. I can remove it and leave a more detailed TODO/NOTE/CHORE comment to look into a more sustainable and/or optimal solution to avoid executing the `.every` on each render.

Reply:
Done! Push to come.